bili.config.ts

```diff
@@ -6,12 +6,17 @@ const config: Config = {
       tsconfigOverride: {
         include: ['src', 'ethers']
       }
-    }
+    },
   },
-
+  extendConfig: (config) => ({
+    ...config,
+    externals: []
+  }),
+  bundleNodeModules: ["nats.ws"],
```

Comment:
@dwojno What do you think of adding a comment here? Something like:
```
nats.ws is bundled because using standard import/require does not work for cjs format. 
nat.ws only provides a .mjs file which cannot be required.
```